rero_mef/marctojson/do_gnd_concepts.py

```diff
@@ -0,0 +1,354 @@
+            }
+        ]
+        if variant_access_point := build_string_list_from_fields(
+            record=self.marc, tag=tag, subfields=subfields, tag_grouping=tag_grouping
+        ):
+            self.json_dict["variant_access_point"] = variant_access_point
+
+    def trans_gnd_relation_pid(self):
+        """Transformation relation pids 682 $0."""
+        if self.logger and self.verbose:
+            self.logger.info("Call Function", "trans_gnd_relation")
+        fields_682 = self.marc.get_fields("682")
+        for field_682 in fields_682:
+            if field_682.get("i") and field_682["i"] == "Umlenkung":
+                subfields_0 = field_682.get_subfields("0")
+                for subfield_0 in subfields_0:
+                    if subfield_0.startswith("(DE-101)"):
+                        self.json_dict["relation_pid"] = {
+                            "value": subfield_0.replace("(DE-101)", ""),
+                            "type": "redirect_to",
+                        }
+
+    def trans_gnd_relation(self):
+        """Transformation relation 551."""
+        if self.logger and self.verbose:
+            self.logger.info("Call Function", "trans_gnd_relation")
+        relations = {}
+        for field_551 in self.marc.get_fields("550"):
+            authorized_aps = set()
+            with contextlib.suppress(Exception):
+                relation_type = "related"
+                if subfield_4 := field_551.get("4"):
+                    if subfield_4 in ["nach", "obal", "obge", "obin"]:
+                        relation_type = "broader"
+                    elif subfield_4[0] in ["vorg"]:
+                        relation_type = "narrower"
+
+                subfields = {"a": ", ", "g": " , "}
+                tag_grouping = [
+                    {
+                        "subtags": "g",
+                        "start": " (",
+                        "end": ")",
+                        "delimiter": "",
+                        "subdelimiter": ", ",
+                    }
+                ]
+                if authorized_ap := build_string_from_field(
+                    field=field_551, subfields=subfields, tag_grouping=tag_grouping
+                ):
+                    relations.setdefault(relation_type, [])
+                    if authorized_ap not in authorized_aps:
+                        authorized_aps.add(authorized_ap)
+                        relations[relation_type].append(
+                            {"authorized_access_point": authorized_ap}
+                        )
+        for relation, value in relations.items():
+            if value:
+                self.json_dict[relation] = value
+
+    def trans_gnd_classification(self):
+        """Transformation classification from field 686."""
+        if self.logger and self.verbose:
+            self.logger.info("Call Function", "trans_gnd_classification")
+        # TODO: find classification
+
+    def trans_gnd_close_match(self):
```

Comment:
You already put the 750 in `identifiedBy`, we should decide if we put it there or in "closeMatch/exactMatch"